### PR TITLE
New Room Free Plugin

### PR DESCRIPTION
**Pull request by gruunday**

### CODE REVIEW

File: steely/plugins/roomcheck.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+def check_room(room):
+    url = get_url(room)
+    # Downloads request from url URL and stores as string in res
+    res = requests.get(url)
+    # Creates Beautiful Soup out of html
+    current_soup = bs4.BeautifulSoup(res.text, "lxml")
+    # Takes all <tr> tags
+    elems = current_soup.select('tr')
+    return elems[14].getText().strip()
```
Comment (byxor) [inline steely/plugins/roomcheck.py]: Pedantic comment from a comment-disliker: These comments are a little repetitive of the source code.

File: steely/plugins/roomcheck.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+__author__ = 'gruunday'
+COMMAND = '.room'
+
+
+def main(bot, author_id, message, thread_id, thread_type, **kwargs):
+    def send_message(message):
+        bot.sendMessage(message, thread_id=thread_id, thread_type=thread_type)
+    if not message:
+        send_message('.room <room number>\nCheck if a dcu room is free right now\n.room <GLA.LG26>')
```
Comment (devoxel) [inline steely/plugins/roomcheck.py]: You should use HELP and just just print the help here.

File: steely/plugins/roomcheck.py [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+        return
+    booking = check_room(message)
+    if booking:
+        send_message(message + ' is not free, there is\n \n' + booking + '\n')
```
Comment (devoxel) [inline steely/plugins/roomcheck.py]: What's with the abundance of newlines here? Also string formatting would be nicer to read here

File: steely/plugins/roomcheck.py [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+    hour = str(date).split()[1][0:2]
+    minute = str(date).split()[1][3:5]
+    if int(minute) > 30:
+        inthour = int(hour)
+        inthour += 1
+    return str(inthour - 8)
+
+
+if __name__ == '__main__':
+    main()
```
Comment (devoxel) [inline steely/plugins/roomcheck.py]: This will just fail immediately, because bot and stuff don't exist

**Pull request merged**